### DISCUSSION TITLE
HTTPS certificate expiring in < 30 days

### DISCUSSION THREAD

**Discussion by eikowagenknecht** [OpenCCU/OpenCCU#2582] [🇺🇸 Q&A] [answered]
### Describe the issue you are experiencing

I got the "critical" warning "HTTPS certificate expiring in < 30 days" on my Raspberrymatic that's installed as a Home Assistant AddOn.

Googling only led me to this thread: https://homematic-forum.de/forum/viewtopic.php?t=51365

But the described "Zertifikat erstellen" button does not seem to exist?

![image](https://github.com/jens-maus/RaspberryMatic/assets/1475672/fd2b22d8-edff-4dd9-8a6b-89762192c39b)

I pressed "Löschen" instead, thinking that maybe it will automatically generate a new one. But the alarm messege still persists. Hopefully I haven't made it worse? I don't see any change in functionality, yet.

### Describe the behavior you expected

Some information on how to fix the error in the error message.

### Steps to reproduce the issue

1. Run the CCU for long enough that the message pops up
2. Wonder what to do
3. ?

### What is the version this bug report is based on?

3.73.9.20231130

### Which base platform are you running?

ha-addon (HomeAssistant Add-on)

### Which HomeMatic/homematicIP radio module are you using?

HmIP-RFUSB

### Anything in the logs that might be useful for us?

```txt
no
```

**Comment by jens-maus** [comment, marked as answer]
Pressing "Löschen" should be the right approach in this situation. After clearing the alarm message it should then not re-appear. Please check and report back.

**Comment by eikowagenknecht** [comment]
Thank you for the response. It seems to have worked, I haven't seen the message since. I still think the instructions (without consulting GitHub issues or the homematic forum) should be clearer.
